packages/ui/src/theme/tokens/components/inAppMessaging/index.ts

```diff
@@ -0,0 +1,8 @@
+// Exporting directly as backdrop ought to eventually become a fully fledged
```

Comment:
I think it would be better to put inside the `inAppMessaging` namespace rather than pre-emptively add it as a primitive. I don't think it would be a breaking change if/when we expose it as a primitive.